### DISCUSSION TITLE
Welcome to Retro Discussions!

### DISCUSSION THREAD

**Discussion by Anjaliavv51** [Anjaliavv51/Retro#75] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.
  For the maintainers, here are some tips 💡 for getting started with Discussions. We'll leave these in Markdown comments for now, but feel free to take out the comments for all maintainers to see.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by Shekhar-Raj** [comment]
<h2>Updated code issue</h2>

Hi Guys! Is anyone facing the same issue that whenever you clone the main repo to your local, then the code in local repo is not updated code?

**Reply by Kiran-F** [reply]
> ## Updated code issue
> Hi Guys! Is anyone facing the same issue that whenever you clone the main repo to your local, then the code in local repo is not updated code?

yeah @Shekhar-Raj I've faced the same issue. and when I made the PR there were some problems in them.

**Comment by Anjaliavv51** [comment]
Please Nominate me for PA!

Google Form Link : https://docs.google.com/forms/d/e/1FAIpQLSdnqJ5crpQFQEQXGTzekfKUxOJCXkSx8AykqvvHzlqBcNGMAQ/viewform

Below are the details to be filled in the form : 
PA Name : Pamidi.Lakshmi Pavananjali 
Repository link of PA : https://github.com/Anjaliavv51/Retro
